packages/recomponents/src/components/r-select/r-select.vue

```diff
@@ -530,6 +534,15 @@
             hasLabel() {
                 return (this.label || '').trim() !== '';
             },
+            computedClasses() {
+                return this.isInvalid ? 'is-error' : '';
+            },
```

Comment:
we have `classes` in other components for classes for root element, let's try to keep naming consistent

Reply:
https://github.com/Rebilly/rebilly-recomponents/pull/56/commits/04160e69b96bb62a2ca0d8e638be4b2c75b32176